src/test/codegen/consts.rs

```diff
@@ -1,4 +1,4 @@
-// compile-flags: -C no-prepopulate-passes
+// compile-flags: -C no-prepopulate-passes -Zmir-opt-level=0
```

Comment:
Is this annotation still needed? It doesn't seem like the difference between `E::A` and `E::B` is big enough for any instantiation for the MIR opt to be triggering.